index.html

```diff
@@ -232,4 +232,36 @@ <h3>Skills</h3>
             </div>
         </section>
     </section>
+    <section class="footer">
+        <div class="header-img-left">
+            <img src="images/image_geometry_1.png" alt="image">
+        </div>
+        <div class="contact">
+            <div class="contact-description">
+                <h3>Get started</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+            </div>
+            <form action="https://formspree.io/f/xbjwgkqr" method="post">
+                <input type="text" name="name" id="name" placeholder="Kiprono" required>
+                <input type="email" name="email" id="email" placeholder="|" value="Your email" required>
+                
+                <textarea name="message" id="msg" cols="30" rows="10"></textarea>
```

Comment:
- [ ]  Please use a placeholder for your message field.

Reply:
resolved